examples/accordion/base.jsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import Accordion from '../../components/accordion'; // `~` is replaced with design-system-react at runtime
+import Panel from '~/components/accordion/private/panel'; // `~` is replaced with design-system-react at runtime
+import IconSettings from '~/components/icon-settings'; // `~` is replaced with design-system-react at runtime
+import Dropdown from '../../components/menu-dropdown'; // `~` is replaced with design-system-react at runtime
```

Comment:
Please use `~/` in examples for doc site compatibility

Reply:
Oof, sorry for the inconsistencies there. I also didn't know the reason we do this is for doc site compatibility. Good to learn.